### DISCUSSION TITLE
About API Routes and SSG

### DISCUSSION THREAD

**Discussion by clovu** [vercel/next.js#66425] [Help]
### Summary

Hey! Thanks for yours efforts! I've encountered a problem: I want to use Next.js API Routes as a mock data source to facilitate front-end development and testing while the backend is still under development. However, API Routes cannot be used with the "output: export" option during static export.

My requirement is to exclude API Routes from processing during the build and only generate static pages. Is there any way to achieve this? If this can be done, there would be no need to use other tools to develop a mock server!

**Comment by dylansmith** [comment]
+1
